### ISSUE TITLE
Crash when copying C++ functions to clipboard

### ISSUE THREAD

**Issue by manicombinostics** [lucasg/Dependencies#129]
I'm viewing a list of functions from C++ DLL. The program crashes when I'm trying to copy any of the functions to clipboard. This happens either when the functions are decorated or undecorated.

```
Exception thrown: 'System.Runtime.InteropServices.COMException' in PresentationCore.dll
An unhandled exception of type 'System.Runtime.InteropServices.COMException' occurred in PresentationCore.dll
OpenClipboard Failed (Exception from HRESULT: 0x800401D0 (CLIPBRD_E_CANT_OPEN))
```

By debugging the latest master the crash happens in PeExport.cs line 154:
https://github.com/lucasg/Dependencies/blob/826fda88b1513a4a3ccdca421fe62075323eb28b/DependenciesGui/Models/PeExport.cs#L154

It seems that issue is already fixed elsewhere. I suggest using try-catch similar to:
https://github.com/lucasg/Dependencies/blob/826fda88b1513a4a3ccdca421fe62075323eb28b/DependenciesGui/DependencyExportList.xaml.cs#L58-L64

Also using Clipboard.SetDataObject(param) instead of Clipboard.SetText() fixes the problem, whichever is the correct way to fix the issue. SetText() seems to be more slower for whatever reason.

**Comment by lucasg** [comment]
Ok fixed it, thanks : 994ba58790f88e381f9fc0ac2c8d635106225f01

**Comment by lucasg** [comment]
PS : i don't like that `Clipboard.SetDataObject` is security critical. I'm copying text so I rather use `SetText`
